Parsers/Sysmon/Sysmon-v9.10-Parser.txt

```diff
@@ -64,7 +64,7 @@ SourceHostname = EventDetail.[10].["#text"], SourcePort = EventDetail.[11].["#te
 DestinationIp = EventDetail.[14].["#text"], DestinationHostname = EventDetail.[15].["#text"], DestinationPort = EventDetail.[16].["#text"], DestinationPortName = EventDetail.[17].["#text"]
 | parse RuleName with * 'technique_id=' TechniqueId ',' * 'technique_name=' TechniqueName 
 | parse Hashes with * 'SHA1=' SHA1 ',' * 'MD5=' MD5 ',' * 'SHA256=' SHA256 ',' * 'IMPHASH=' IMPHASH 
```

Comment:
Have to remove the Parse here for hashes otherwise the event is not output and the project fails at the end.